aws/src/integration/java/org/apache/iceberg/aws/AwsIntegTestUtil.java

```diff
@@ -127,6 +129,39 @@ public static void cleanS3Bucket(S3Client s3, String bucketName, String prefix)
     }
   }
 
+  public static void cleanS3ExpressBucket(S3Client s3, String bucketName, String prefix) {
+    // For S3 express we just need to delete the objects not the versions.
+    ListObjectsV2Request listRequest =
+        ListObjectsV2Request.builder()
+            .bucket(bucketName)
+            .prefix(prefix)
+            .build();
+
+    ListObjectsV2Iterable paginatedListResponse = s3.listObjectsV2Paginator(listRequest);
+    List<ObjectIdentifier> objectsToDelete = Lists.newArrayList();
+    paginatedListResponse.contents().stream()
+        .forEach(s3Object -> {
+          if (objectsToDelete.size() == BATCH_DELETION_SIZE) {
+            deleteObjects(s3, bucketName, objectsToDelete);
```

Comment:
this is potentially a good implicit test for `SupportsBulkOperations.deleteFiles()` which, being used in production, is something to stress as much as possible